doc/taskApi.yml

```diff
@@ -0,0 +1,221 @@
+swagger: '2.0'
+info:
+  description: Tasks API
+  version: 1.0.0
+  title: Tasks API
+basePath: /act.dev
+schemes:
+  - http
+  - https
+paths:
+  /taoTaskQueue/RestTask/get:
+    get:
+      tags:
+        - Get Task
+      summary: Get Task
+      description: Get Task
+      operationId: GetTask
+      parameters:
+        - name: taskId
+          in: query
+          description: Task id to delete
+          required: true
+          type: string
+          format: int64
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            properties:
+              success:
+                type: boolean
+              data:
+                $ref: '#/definitions/dataItem'
+              errorCode:
+                type: string
+              errorMsg:
+               type: string
+        '400':
+          description: Bad request if you send invalid parameters.
+        '401':
+          description: Unauthorized
+          schema:
+            $ref: '#/definitions/errorModel'
+  /taoTaskQueue/RestTask/getAll:
```

Comment:
I would use some limit option here. I guess this one would be used to get all task for a user. Don't we really want to use any pagination?

Reply:
Yes, I was thinking about this, also, I will add pagination options